responses/responses.json

```diff
@@ -197,6 +197,24 @@
     "list_of_words": ["i", "sleepy"],
     "is_single_response": false,
     "required_word": []
+  },
+
+  "number": {
+    "response": [
+      "I don't have any phone number for you cuz I'm a bot."
+    ],
+    "list_of_words": ["can","i","have","your","number"],
+    "is_single_response": false,
+    "required_word": []
+  },
+
+  "siblings": {
+    "response": [
+      "Um... I don't know about that dad didn't tell me about that at all."
```

Comment:
I think this should be "Um... I don't know about that much... My dad didn't tell me about that. Maybe one day I'll ask him."

Reply:
And also, I think of you can make more responses here, this would be better.